LPS-92663 Fix staged model listener

matethurzo opened the pull request.

Commit:
LPS-93201 Add table

lipusz committed:
LPS-91813 Update netcdf.jar version to 4.3.23 to fix NoClassDefFoundError during reindex when uploading a HDF5 file

Contributed solution from @bselack

https://issues.liferay.com/browse/LPS-91813

lipusz committed:
LPS-91813 Update versions-ext.xml

lipusz committed:
LPS-91813 Run ant build-lib-versions

Commit:
LPS-90260 product-navigation - Add private page to current site

Commit:
LPS-92519 product-navigation - Add PersonalMenuConfigurationTracker

Commit:
LPS-92519 product-navigation - Add configuration for showing the personal menu in the control menu

Commit:
LPS-92518 product-navigation - Show the personal menu in the control menu

ealonso committed:
LPS-93184 Reorder options

ealonso committed:
LPS-93184 Adapt styles to work in small columns

dgarciasarai committed:
LPS-92803 Add new method

Commit:
LPS-90260 Set inheritance option for classes extending PersonalMenuConfigurationForm

Commit:
LPS-91408 reload after modal close

Commit:
LPS-92518 product-navigation-user - Deprecate user product menu entries

Hanlf committed:
LRQA-47716 Use right locater to view username

wincent committed:
LPS-93162 Fixes segments/segments-web tests

Somewhat confusing to see a couple of legit looking failures here before
this commit, given that I saw these green the other day in f73bfed5d8b6.
Nevertheless, there was a clear prop-type violation here, a failure to
pass a necessary array prop that later led the test to explode when the
component tried to call `find()` on `undefined`, and some stale
snapshots. I can't see any recent changes to this directory that would
have broken the tests; maybe what I saw the other day was a false
negative.

In any case, I made the necessary updates and now `yarn test` is green.

alee8888 committed:
LPS-92368 Add option to display data types for all selection Custom Field input types

alee8888 committed:
LPS-92368 Update getters to take into account for data arrays

dgarciasarai committed:
LPS-92803 Regen

Commit:
LRDOCS-6454 edit javadoc

Commit:
LPS-92518 product-navigation-personal-menu-web - Fix gradle dependencies

Commit:
LRDOCS-6154 edit javadoc

Preston-Crary committed:
LRDOCS-6454 This transaction is configured, not the other way around.

pei-jung committed:
LPS-92518 Do not namespace this. This is a special container used by ClayDropdown for the dropdown items

pei-jung committed:
LPS-92518 Rename var and update CSS

pei-jung committed:
LPS-92518 Move package

dgarciasarai committed:
LPS-92803 Use new method in content-list API

pavel-savinov committed:
LPS-92663 Check if layout instance is head instance before execution of related logic

Commit:
LPS-92663 All the layout model listeners were designed for the resource entity, so adding a clause omitting the versions

darquesdev committed:
LPS-93201 Implement local service

Commit:
LPS-92663 Avoid NPE

darquesdev committed:
LPS-93201 Regen

Commit:
LPS-91408 - Source formatting

Commit:
LPS-93163 Use new popover

victorlaerte committed:
LPS-91859 Open Api Changes

victorlaerte committed:
LPS-91859 buildRest

victorlaerte committed:
LPS-91859 Add FormStructure Resource

Commit:
LRDOCS-6454 more javadoc edits

Commit:
LPS-93219 Use parameter id

dgarciasarai committed:
LPS-93312 Change ordering buttons in Asset Publisher

dgarciasarai committed:
LPS-93312 Change ordering buttons in Content Sets

dgarciasarai committed:
LPS-93312 Remove unused method

brianchandotcom committed:
LPS-91859 Simplify

michaelhashimoto committed:
LRCI-101 Add invocation params for testray & branch info to testsuite upstream jobs

michaelhashimoto committed:
LRCI-101 Add JenkinsResultsParserUtil#getRemoteURL for display purposes

pei-jung committed:
LPS-92519 Rename to 'showInControlMenu' to keep the naming consistent with other configuration methods

michaelhashimoto committed:
LRCI-101 Make cohort invocation configurable by a environment variable

brianchandotcom committed:
LPS-91859 Use _portal for consistency

brianchandotcom committed:
LPS-91859 Use proper name

brianchandotcom committed:
LPS-91859 Leave it in *ResourceImpl if it does not have to be reused

brianchandotcom committed:
LPS-91859 Inline

pei-jung committed:
LPS-92519 Build lang

michaelhashimoto committed:
LRCI-101 Check if invoked job is actually still running

Preston-Crary committed:
LPS-92016 Avoid 'org.osgi.enterprise' artifact.

Hanlf committed:
LRQA-47879 Update locator of MediaGallery#IMAGE_TITLE

petershin committed:
LPS-93258 Enable the registry flag for Yarn

petershin committed:
LPS-93258 Enable the production flag for Yarn

petershin committed:
LPS-93258 Change task type to ExecuteNpmTask

petershin committed:
LPS-93258 Update hostname in yarn.lock (see https://github.com/yarnpkg/yarn/issues/5892)

petershin committed:
LPS-93258 Disable external update check https://yarnpkg.com/latest-version

petershin committed:
LPS-93258 Use latest gradle-plugins-node

petershin committed:
LPS-93258 Use latest gradle-plugins-jsdoc

petershin committed:
LPS-93258 Use latest gradle-plugins-js-transpiler

petershin committed:
LPS-93258 Use latest gradle-plugins-js-module-config-generator

petershin committed:
LPS-93258 Use latest gradle-plugins-gulp

Hanlf committed:
LRQA-48008 Update locator of Icon#MOVE

Hanlf committed:
LRQA-47884 Go back to Folder before viewing DM

pei-jung committed:
LPS-92519 Rename

JoyceWang08211 committed:
LRQA-47723 Update macro to use capital letter for value

Hanlf committed:
LRQA-47878 Update locator of ItemSelector#SELECT_FILE_IMAGE_PREVIEW

ruben-pulido committed:
LPS-87529 Rename content list to content set in rest-openapi.yaml

ruben-pulido committed:
LPS-87529 Generate

JoyceWang08211 committed:
LRQA-47721 Update case to expand the pages section

Hanlf committed:
LRQA-47906 Add locator of BREADCRUMB_FOLDER

Hanlf committed:
LRQA-47906 Navigate to Home to choose folder

Commit:
LPS-92663 Fix staged model listener

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 committed:
LPS-93106 Show text properties also in editables mapped to specific content

ruben-pulido committed:
LPS-87529 Move ContentListElementResourceImpl code to ContentSetElementResourceImpl

ruben-pulido committed:
LPS-87529 Delete content list classes not removed by the generator

ruben-pulido committed:
LPS-87529 Add by-key and by-uuid endpoints for content-sets to yaml file

4lejandrito committed:
LPS-93274 Simplifying sharing company settings

jbalsas committed:
LPS-93267 Removes autobind decorator from dynamic-data-mapping-form-web

jbalsas committed:
LPS-93267 Removes unnecessary configuration overrides in dynamic-data-mapping-form-web

Commit:
LPS-93162 test command back

ealonso committed:
LPS-93213 Check if the assetEntryUsagesList exist

jbalsas committed:
LPS-93267 Removes unnecessary polyfill dependencies and simplify build step in dynamic-data-mapping-form-web

Commit:
LPS-92663 One more error

jbalsas committed:
LPS-93267 Removes autobind decorator from dynamic-data-mapping-form-field-type

Commit:
LPS-93193 Add Array.of polyfill for IE11

ruben-pulido committed:
LPS-87529 Generate

ruben-pulido committed:
LPS-87529 Add methods to fetch asset list entries by uuid and key to
asset list entry service and local service

Commit:
LPS-93272 Support managing content set variations

ruben-pulido committed:
LPS-87529 Generate

ruben-pulido committed:
LPS-87529 Add methods to fetch asset list entries by uuid and key to ContentSetElementResourceImpl

Commit:
LPS-92663 One more

jbalsas committed:
LPS-93267 Fixes jsp path

jbalsas committed:
LPS-93267 Removes unnecessary dependencies and simplifies build step in dynamic-data-mapping-form-field-type

jbalsas committed:
LPS-93267 Removes autobind decorator from dynamic-data-mapping-form-builder

Commit:
LPS-92663 More

jbalsas committed:
LPS-93267 Removes unnecessary polyfill dependencies and simplify build step in dynamic-data-mapping-form-builder

jbalsas committed:
LPS-93267 Removes unnecessary configuration overrides in dynamic-data-mapping-form-builder

darquesdev committed:
LPS-92855 Remove get default segments experience method

darquesdev committed:
LPS-92855 Replace use of default segments experience for constant

ruben-pulido committed:
LPS-87529 Add getAssetListEntry(assetListEntryId) method to AssetListEntryServiceImpl

ruben-pulido committed:
LPS-87529 Generate

ruben-pulido committed:
LPS-87529 Call getAssetListEntry instead of fetchAssetListEntry so a 404 error is
returned instead of 500, when the content set requested does not exist

darquesdev committed:
LPS-92855 Remove addition of default segments experience method

darquesdev committed:
LPS-92855 Remove unused exceptions and references

darquesdev committed:
LPS-92855 Fix tests

darquesdev committed:
LPS-92855 Regen

jbalsas committed:
LPS-93289 Replaces lodash.uniqueId dependency with metal

jbalsas committed:
LPS-93289 Replaces lodash.debounce dependency with metal

pavel-savinov committed:
LPS-93081 Maintain page template name for associated layout, don't use class type name

jbalsas committed:
LPS-93289 Excludes unnecessary dependencies from being bundled

Commit:
LPS-93272 Build lang

Commit:
LPS-93272 Fix segment name

victorlaerte committed:
LPS-91859 inline

Commit:
LPS-93219 Add permissions to data records

adolfopa committed:
LPS-90739 Use same action name for categories and subcategories

4lejandrito committed:
LPS-93307 Adding sharing demo

Commit:
LPS-93267 Also removes autobind decorator from this file

sandrodw3 committed:
LPS-93212 Modified condition for adding target attribute in edit mode

sandrodw3 committed:
LPS-93212 Modified condition for adding target attribute in view mode

darquesdev committed:
LPS-92934 Create new table during upgrade process since service builder will not add it

Commit:
LPS-93267 Makes CSF happy again

victorg1991 committed:
LPS-93299 Add exception mappers for KnowledgeBase

victorg1991 committed:
LPS-93299 Fix PUT

Commit:
LPS-92855 Same pattern as for default segment soy context

Commit:
LPS-92855 SF

victorg1991 committed:
LPS-93309 Validate name on update

pavel-savinov committed:
LPS-91753 Add appropriate message when publishing layout, page template or display page

pavel-savinov committed:
LPS-91753 Show success message when publishing layout, page template or display page

pavel-savinov committed:
LPS-91753 Missing language keys

pavel-savinov committed:
LPS-91753 Buildlang

jbalsas committed:
LPS-93292 Excludes unnecessary dependencies from frontend-taglib-chart

ealonso committed:
LPS-91753 SF

ealonso committed:
LPS-91753 Reorder

4lejandrito committed:
LPS-93309 Using StringUtil to compare strings

pyoo47 committed:
LRCI-36 Fix PMD error

4lejandrito committed:
LPS-93309 Adding test

ealonso committed:
LPS-93272 Update styles

petershin committed:
COMMERCE-1250 Add additional check for 202 response

adolfopa committed:
LPS-92670 Force version customization on check in

adolfopa committed:
LPS-92670 When checked out, add field as hidden input

We need to include the field in the request so that the backend knows the user
is specifying an explicit version increment.

ealonso committed:
LPS-93272 Fix redirect

Commit:
LPS-77699 Revert Language_en.properties changes made by translation service

brianchiu committed:
LPS-92248 Fix values that were changed mistakenly

Commit:
LPS-93267 Fix unit tests

michaelhashimoto committed:
LRCI-101 Print a description when a testsuite upstream job was not triggered

michaelhashimoto committed:
LRCI-101 Retire previous builds that exceed a certain age

brianchandotcom committed:
LPS-93213 only used once, no need to assign

brianchandotcom committed:
LPS-93312 Sort

brianchandotcom committed:
LPS-93163 SF

Commit:
LPS-91859 Inline what's not reused

Preston-Crary committed:
LRDOCS-6154 More clear.

brianchandotcom committed:
LPS-92934 SF

Preston-Crary committed:
LRDOCS-6454 More clear.

joshchong committed:
LRQA-48040 Quarantine on search suites

brianchandotcom committed:
LPS-92803 SF

brianchandotcom committed:
LPS-92855 This part makes no sense because you're setting segmentsEntryIds, even though you're returning segmentsExperienceIds, so.. just inline it

brianchandotcom committed:
LPS-93258 prep next

brianchandotcom committed:
LPS-91753 Sort

brianchandotcom committed:
LPS-93272 SF

brianchandotcom committed:
LPS-93106 Sort

brianchandotcom committed:
LPS-92519 Hugo SF

brianchandotcom committed:
LPS-92519 Only used in one place

brianchandotcom committed:
LPS-92519 SF

jpince committed:
LRQA-48046 Remove old projects

joshchong committed:
LRQA-48047 Quarantine AssetPublisherExportImportTest on search test suite

pei-jung committed:
LPS-92518 Don't inline the special container to avoid messing up with the other dropdown menus. Detach dropdown menu on click instead

Commit:
LPS-93335 Adjust Data Layout portlet names

Commit:
LRDOCS-6454 format javadoc

victorg1991 committed:
LPS-93299 Add required properties for KB

victorg1991 committed:
LPS-93299 Autogenerated

brianchandotcom committed:
LPS-93312  we need this import to compile

brianchandotcom committed:
Revert "LPS-93312  we need this import to compile"

This reverts commit 54c379b1afa7f8972e92a5dfcb466455c9bbd915.

brianchandotcom committed:
Revert "LPS-93312 Remove unused method"

This reverts commit 632d7a7db565721dc81d830f73a346053e010e09.

victorg1991 committed:
LPS-93264 Fix exception messages

brianchandotcom committed:
LPS-93309 match headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/jaxrs/exception/mapper/DuplicateFriendlyURLEntryExceptionMapper.java

victorg1991 committed:
LPS-93264 Validator.isNotNull returns false for empty string, but we may want to patch the description to an empty string, so we cannot rely on that method

victorg1991 committed:
LPS-93264 Autogenerated

brianchandotcom committed:
LPS-93309 Simplify

brianchandotcom committed:
LPS-93264 SF

victorg1991 committed:
LPS-93299 Do not allow empty file

brianchandotcom committed:
LPS-93299 Wordsmith, I typed: git grep -A 5 "binaryFile ==" modules/apps/headless/headless-delivery

brianchandotcom committed:
LPS-93264 never used now

brianchiu committed:
LRQA-47973 Update web content selected structure locator

brianchiu committed:
LRQA-47973 Need to assert partial text

brianchiu committed:
LRQA-47973 Structure and template are in separate panels now

brianchiu committed:
LRQA-47973 Update selected template field locator

ruben-pulido committed:
LPS-87529 Take segments into account after solving rebase conflicts

Preston-Crary committed:
LRDOCS-6154 SF.

pei-jung committed:
LPS-90261 Add configuration option to set the personal app's look and feel

patriciajeanperez committed:
LRQA-48041 Use different macro for clicking on ellipsis menu item

brianchiu committed:
LRQA-47973 Fix for localization tests

pyoo47 committed:
LRCI-101 Rename

brianchiu committed:
LRQA-47973 Details panel no longer exists in web content

brianchiu committed:
LRQA-47973 Remove instances of tests changing structure mid edit

patriciajeanperez committed:
LRQA-48038 There's no more dropdown in sidebar

brianchiu committed:
LRQA-47906 Fix poshi validation

patriciajeanperez committed:
LRQA-48038 Update macro for clicking on Version link in DM sidebar

pei-jung committed:
LPS-90261 Build lang

patriciajeanperez committed:
LRQA-48038 Macro not needed anymore

austinchiang committed:
LRQA-48052 Remove Navigation Links assertion

patriciajeanperez committed:
LRQA-48038 Use macro for asserting version

patriciajeanperez committed:
LRQA-48038 Fix locator

pei-jung committed:
LPS-92368 Update precision options for number selection types

patriciajeanperez committed:
LPS-90739 Add option for category description

patriciajeanperez committed:
LPS-90739 Use macro that adds a category

brianchandotcom committed:
LPS-90261 wordsmith

brianchandotcom committed:
LPS-90261 regen

pei-jung committed:
LPS-92368 Simplify

brianchandotcom committed:
LRCI-101 prep next

brianchandotcom committed:
LPS-87529 assetListEntryLocalService.get* throws an exception if it's not found, assetListEntryLocalService.fetch* returns null if it does not exist

brianchandotcom committed:
LPS-87529 Like this

pei-jung committed:
LPS-92368 Reuse code when possible to avoid duplicate code

pei-jung committed:
LPS-92368 Return correct number type

brianchandotcom committed:
LPS-87529 Rename

brianchandotcom committed:
LPS-91859 only used once

brianchandotcom committed:
LPS-91859 rename

brianchandotcom committed:
LPS-91859 rename

brianchandotcom committed:
LPS-91859 Don't chain on stream, just on the stuff after

brianchandotcom committed:
LPS-91859 same thing

brianchandotcom committed:
LPS-91859 rename

brianchandotcom committed:
LPS-91859 Group together

brianchandotcom committed:
LPS-91859 Group together

brianchandotcom committed:
LPS-91859 Group together

brianchandotcom committed:
LPS-91859 Chain after stream, not on stream

brianchandotcom committed:
LPS-91859 Group together

brianchandotcom committed:
LPS-91859 Group together

brianchandotcom committed:
LPS-91859 Group together

brianchandotcom committed:
LPS-91859 sort

brianchandotcom committed:
LPS-91859 consistency

brianchandotcom committed:
LPS-91859 sort

brianchandotcom committed:
LPS-91859 Consistency

brianchandotcom committed:
LPS-91859 Group together

brianchandotcom committed:
LPS-91859 Buy space

brianchandotcom committed:
LPS-91859 Rename

brianchandotcom committed:
LPS-91859 match name in DDMFormField

brianchandotcom committed:
LPS-91859 Good enough for now

brianchandotcom committed:
LPS-91859 Rename

brianchandotcom committed:
LPS-91859 auto SF

brianchandotcom committed:
LPS-91859 rename

brianchandotcom committed:
LPS-93307 optimize for readability over performance

brianchandotcom committed:
LPS-93307 Sort

brianchandotcom committed:
LPS-93307 SF

brianchandotcom committed:
LPS-93307 Simplify

JoyceWang08211 committed:
LRQA-48030 Update case to declare portlet name

JoyceWang08211 committed:
LRQA-48032 Update macro to make the edit link visible

brianchandotcom committed:
Record reference to liferay-portal/master-private with git-commit-portal-private as ef0aa705d8b6baff3b164b2930e6f4ec12b03bdd
